01who_we_are/02values.md

```diff
@@ -7,31 +7,73 @@ We take our core values very seriously. They're the underpinning for what we do,
+
+Some examples of this value in action are:
+
+- Our regular process of peer review for our code and content, where we provide feedback and encourage for improvement.
+- Contributing our standard high quality videos for free to enhance the Drupal community documentation.
 
 **Feed creativity**
 
-Creativity creates space for innovative ideas and inventive solutions. We embrace the full creative process, which includes failing, learning, and trying again.
+Creativity creates space for innovative ideas and inventive solutions. We embrace the full creative process, which includes failing, learning, and trying again, in a safe and respectful environment. We are open to creative change and are not afraid to change existing things as well as creating something new. Through openly sharing our new ideas in the wider world we aim to also foster creativity in others.
+
+Some examples of this value in action are:
+
+- We explore new ideas for how to deliver our content, from written tutorials to new hands-on project guides. Ideas start from customer feedback and result in features like the ability to access a pre-made site sandbox to avoid customers needing to set up a new site.
+- We re-evaluated our business strategy and decided to change the company name to better reflect our vision. 
+
 
 **Spread happiness**
 
-We believe in the power of happiness. We encourage playfulness, positivity, and kindness in everything we do.
+We believe in the power of happiness. We encourage playfulness, positivity, and kindness in everything we do. We try not to take ourselves too seriously and recognize that being approachable and kind makes learning more fun and less stressful. We are  aware of the impact our attitude has on the people around us and that impact spreads far beyond our personal and professional interactions.
+
+Some examples of this value in action are:
+
+- Creating fun examples in our learning materials, like the ice cream shop example for learning plugins or the batman cartoon for events.
+- Fun stickers! Our sparkly Drupal ponies continue to be a favorite even today.
+- We respond to customer support timely and cheerfully, even the grumpy ones. We're human too, and we get that people have bad days. We like to try to make them a little better when we can.
+
 
 **Work together**
 
-We keep open minds and listen to each other to find common ground and consensus. We support and help each other in ways that make our entire team stronger.
+We believe in Open Source, not only in open software but also in the principles of collaboration and sharing. We fully engage with Open Source communities through contribution and have incorporated these fundamentals throughout our company. We keep open minds and listen to each other to find common ground and improve each other’s work without judgment. We apply these same principles when working with our customers to solve their problems.
```

Comment:
I think documentation could be important to mention here too, since it is such a big part of OS, and how we function as a company

Reply:
I'm not sure how exactly to weave that in. I don't want to put undo attention on documentation in terms of contributions because we contribute in many ways that don't involve documentation at all.